tests/alive-tv/attrs/noundefaggr-disableundefinput.srctgt.ll

```diff
@@ -0,0 +1,33 @@
+; TEST-ARGS: -disable-undef-input
+; FIXME: This transformation should raise 'Source is more defined than target' error
```

Comment:
uhm, so the issue is that padding should always be poison irrespective of the disable-xx-input, right?  We can do that, as we know which fields are padding and which are not, right?